pretyping/unification.ml

```diff
@@ -417,7 +417,7 @@ let default_no_delta_unify_flags ts =
 
 let allow_new_evars sigma =
   let undefined = Evd.undefined_map sigma in
-  AllowFun (fun evk -> not (Evar.Map.mem evk undefined))
+  AllowedEvars.except @@ Evar.Map.domain undefined
```

Comment:
This is slow and should be replaced by the predicate function.

Reply:
Right, will fix.

Reply:
Done.